_templates/how-to.md

```diff
@@ -1,26 +1,72 @@
 ---
 title: [Character Limit: 255, Recommendation is to have <= 75 characters for a title as this can impact indexing by search engines like Google, Bing, etc. The article title should be a combination of the main problem statement and if applicable, the key environment statement. Also determines the title in the table of contents.]
-description: [Brief description of the How-To article]
-type: [HowTo]
-page_title: [Text that will populate the HTML file’s head:title attribute]
+description: [Character Limit: 255, Recommendation is to have <= 155 characters for a description as this can impact indexing by search engines like Google, Bing, etc. The description should be brief overview of the how-to article. It should exclude unnecessary phrases like "This article contains ..."]
+type: [how-to]
+page_title: [Text that will populate the HTML file’s head:title attribute. Normally, this should be the same as the title field.]
```

Comment:
Well, perhaps there has been some misunderstanding. The usual advice for page title and article title is that they differ in some way in order to cover more key words and be more discoverable via searches. This is a recommendation and not a requirement though. If we do decide for the two titles to be identical, we can perhaps handle this via the layouts so that you don't need to explicitly specify this field.

My suggestion is to discuss this at our weekly meeting and come up with a decision once and for all (including whether and what system information we'll be showing).